pyo3-ffi/src/cpython/pyframe.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(any(PyPy, Py_3_11))]
```

Comment:
Is this available in PyPy? I couldn't find anything about it.

Reply:
I couldn't find it either :sweat_smile: 

I've added `#[cfg(not(PyPy))]` to the `pub(crate) mod pyframe;` line on `src/cpython/mod.rs` because PyPy does not have something like "pyframe.h" in their source code.

Commit: https://github.com/PyO3/pyo3/pull/3500/commits/4931b00cbc8491f4562b76f13266ec127a017529